packages/wrapped-keys/src/lib/types.ts

```diff
@@ -101,6 +101,31 @@ export interface StoreEncryptedKeyResult {
   pkpAddress: string;
 }
 
+/** Properties required to persist a batch of encrypted keys into the wrapped-keys backend storage service
+ *
+ * @typedef StoreEncryptedKeyBatchParams
+ * @extends BaseApiParams
+ *
+ */
+export type StoreEncryptedKeyBatchParams = BaseApiParams & {
+  keyBatch: Pick<
+    StoredKeyData,
+    'publicKey' | 'keyType' | 'dataToEncryptHash' | 'ciphertext' | 'memo'
+  >[];
+};
+
+/** Result of storing a batch of private keys in the wrapped keys backend service
+ * Includes an array of unique identifiers, which are necessary to get the encrypted ciphertext and dataToEncryptHash in the future
+ *
+ * @typedef StoreEncryptedKeyBatchResult
+ * @property { string } pkpAddress The LIT PKP Address that the key was linked to; this is derived from the provided pkpSessionSigs
+ * @property { string[] } ids Array of the unique identifiers (UUID V4) of the encrypted private keys in the same order provided
+ */
+export interface StoreEncryptedKeyBatchResult {
+  ids: string[];
```

Comment:
Are the ids used by the user in any way?

Reply:
The _end-user_ generally would never see them -- they would see the `memo` field and probably public key -- the `id` is only used by developers when querying specific keys encrypted data :+1: